apps/backend/src/libs/modules/encryption/encryption.service.ts

```diff
@@ -0,0 +1,55 @@
+import bcrypt from "bcrypt";
+
+import { ApplicationError } from "~/libs/exceptions/exceptions.js";
+
+import { EncryptionServiceMessage } from "./libs/enums/enums.js";
+
+const DEFAULT_SALT_ROUNDS = 10;
+
+class EncryptionService {
```

Comment:
Let's name it encryption, and it should have the same structure as the logger module, with BaseEncryption class
```suggestion
class BaseEncryption implement Encryption {
```